src/main/java/com/example/squiz/exceptions/customExceptions/QuizNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.example.squiz.exceptions.customExceptions;
```

Comment:
Maybe we can use generic Not Found exception instead of Quiz Not Found exception